target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-whw03v2.dts

```diff
@@ -0,0 +1,528 @@
+            qcom,wifi_noc_memtype_m0_m2 = <TCSR_WIFI_NOC_MEMTYPE_M0_M2>;
+        };
+    };
+
+
+    keys {
+        compatible = "gpio-keys";
+
+        reset {
+            label = "reset";
+            gpios = <&tlmm 18 GPIO_ACTIVE_LOW>;
+            linux,code = <KEY_RESTART>;
+        };
+    };
+};
+
+
+&tlmm {
+    mdio_pins: mdio_pinmux {
+        mux_1 {
+            pins = "gpio6";
+            function = "mdio";  
+            bias-pull-up;
+        };
+
+        mux_2 {
+            pins = "gpio7";
+            function = "mdc";
+            bias-pull-up;
+        };
+    };
+
+    i2c_0_pins: i2c_0_pinmux {
+        mux {
+            function = "blsp_i2c0";
+            pins = "gpio20", "gpio21";
+            bias-disable;
+        };
+    };
+
+    serial_0_pins: serial0_pinmux {
+        mux {
+            pins = "gpio16", "gpio17";
+            function = "blsp_uart0";
+            bias-disable;
+        };
+    };
+
+    serial_1_pins: serial1_pinmux {
+        mux {
+            pins = "gpio8", "gpio9", "gpio10", "gpio11";
+            function = "blsp_uart1";
+            bias-disable;
+        };
+    };
+
+    spi_0_pins: spi_0_pinmux {
+        mux {
+            function = "blsp_spi0";
+            pins = "gpio13", "gpio14", "gpio15";
+            drive-strength = <12>;
+            bias-disable;
+        };
+
+        mux_cs {
+            pins = "gpio12";
+            drive-strength = <2>;
+            bias-disable;
+            output-high;
+        };
+    };
+    
+    spi_1_pins: spi_1_pinmux {
+        mux_1 {
+            function = "blsp_spi1";
+            pins = "gpio44", "gpio46","gpio47";
+            bias-disable;
+        };
+
+        mux_2 {
+            pins = "gpio31", "gpio45", "gpio49";
+            function = "gpio";
+            bias-pull-up;
+            output-high;
+        };
+
+        mux_3 {
+            pins = "gpio42";
+            function = "gpio";
+            input;
+        };
+    };
+
+    wifi_0_pins: wifi0_pinmux {
+        // Bluetooth coexist pin
+        mux {
+            bias-pull-up;
+            drive-strength = <6>;
+            function = "gpio";
+            output-high;
+            pins = "gpio52";
+        };
+    };
+
+    
+    zigbee_0 {
+        gpio-hog;
+        gpios = <29 GPIO_ACTIVE_HIGH>;
+        bias-disable;
+        output-low;
+    };
+
+    zigbee_1 {
+        gpio-hog;
+        gpios = <50 GPIO_ACTIVE_HIGH>;
+        bias-disable;
+        input;
+    };
+
+    bluetooth_enable {
+        gpio-hog;
+        gpios = <32 GPIO_ACTIVE_HIGH>;
+        output-high;
+    };
+};
+
+&mdio {
+    status = "okay";
+    pinctrl-0 = <&mdio_pins>;
+    pinctrl-names = "default";
+    phy-reset-gpios = <&tlmm 19 GPIO_ACTIVE_LOW>;
+};
+
+&ethphy0 {
+    status = "disabled";
+};
+
+&ethphy1 {
+    status = "disabled";
+};
+
+&ethphy2 {
+    status = "disabled";
+};
+
+&ethphy3 {
+    status = "okay";
+    reg = <0x1b>;
+    #reg = <0x03>;
```

Comment:
These commented out properties can be dropped.

`status` can be dropped as well as they are enabled by default

Reply:
done